### DISCUSSION TITLE
Copilot 1.190.0 not working in Company Proxy

### DISCUSSION THREAD

**Discussion by narikotomo91** [community/community#123734] [Copilot Conversations] [answered]
### Select Topic Area

Bug

### Body

Hello everyone,
I am currently using GitHub Copilot on VS Code within my company's network and through the company proxy. The previous versions of GitHub Copilot on VS Code (Copilot 1.88.0) worked fine. However, after updating to version 1.90.0, I encountered an issue where Copilot is no longer usable.

My company's proxy requires a username and password. Previously, I was using Helix Fetcher without any problems, but after the update, I encountered an error.
Below are the details of the error.

![image](https://github.com/community/community/assets/110339373/d326d5f2-7e8b-411a-91d8-e7310d2011c0)

proxy config:

![image](https://github.com/community/community/assets/110339373/78c6bded-f3d2-42c2-8af1-a7905e310e10)

Diagnostics (Secret information replaced)

`## Copilot

- Version: 1.190.0
- Build: prod
- Editor: vscode/1.89.1

## Environment

- http_proxy: http://username:pass@proxy-ip:8080
- https_proxy: n/a
- no_proxy: n/a
- SSL_CERT_FILE: n/a
- SSL_CERT_DIR: n/a
- OPENSSL_CONF: n/a

## Node setup

- Number of root certificates: 141
- Operating system: Windows_NT
- Operating system version: 10.0.19045
- Operating system architecture: x64
- NODE_OPTIONS: n/a
- NODE_EXTRA_CA_CERTS: n/a
- NODE_TLS_REJECT_UNAUTHORIZED: n/a
- tls default min version: TLSv1.2
- tls default max version: TLSv1.3

## Network Configuration

- Proxy host: <proxy-ip>
- Proxy port: 8080
- Kerberos SPN: n/a
- Reject unauthorized: disabled
- Fetcher: ElectronFetcher

## Reachability

- github.com: HTTP 407 - authenticationrequired
- copilot-proxy.githubusercontent.com: HTTP 407 - authenticationrequired
- api.githubcopilot.com: HTTP 407 - authenticationrequired
- default.exp-tas.com: HTTP 200 - 

## VS Code Configuration

- HTTP proxy: http://username:pass@proxy-ip:8080
- HTTP proxy authentication: n/a
- Proxy Strict SSL: false
- Extension HTTP proxy support: override

## Extensions

- Is `win-ca` installed?: false
- Is `mac-ca` installed?: false

## Authentication

- GitHub username: hieunt8991`

**Comment by devm33** [comment, marked as answer]
Sorry about this. Please update to 1.192.0 to resolve this issue.
